vite.config.mts

```diff
@@ -30,17 +30,38 @@ export default defineConfig({
         dts({
             include: ['src'],
         }),
-        externalizeDeps({
-            include: [/^react-is(?:\/.*)?$/, /^@mui(?:\/.*)?$/],
-        }),
     ],
     build: {
         lib: {
             entry: path.resolve(__dirname, 'src/index.js'),
             formats: ['es'],
         },
         rollupOptions: {
+            external: (id: string) =>
+                !id.startsWith('.') && !path.isAbsolute(id),
+            // We do this to keep the same folder structure
+            // from https://rollupjs.org/configuration-options/#input
+            input: Object.fromEntries(
+                globSync('src/**/*.{js,jsx,ts,tsx}', {
+                    ignore: [
+                        'src/index.d.ts',
+                        'src/vite-env.d.ts',
+                        'src/**/*.test.{js,jsx,ts,tsx}',
+                    ],
+                }).map((file) => [
+                    // This remove `src/` as well as the file extension from each
+                    // file, so e.g. src/nested/foo.js becomes nested/foo
+                    path.relative(
+                        'src',
+                        file.slice(0, file.length - path.extname(file).length)
+                    ),
+                    // This expands the relative paths to absolute paths, so e.g.
+                    // src/nested/foo becomes /project/src/nested/foo.js
+                    url.fileURLToPath(new URL(file, import.meta.url)),
+                ])
+            ),
             output: {
+                chunkFileNames: 'chunks/[name].[hash].js', // we have few remaining chunks, couldn't remove them
```

Comment:
strange I don't have chunks anymore in the build despite the lib-inject-css

<details>
  <summary>expand</summary>
  
```shell
$ tree dist
dist
├── components
│   ├── AuthenticationRouter
│   │   ├── AuthenticationRouter.d.ts
│   │   ├── AuthenticationRouter.js
│   │   ├── index.d.ts
│   │   └── index.js
│   ├── CardErrorBoundary
│   │   ├── card-error-boundary.d.ts
│   │   ├── card-error-boundary.js
│   │   ├── index.d.ts
│   │   └── index.js
│   ├── DirectoryItemSelector
│   │   ├── directory-item-selector.d.ts
│   │   └── directory-item-selector.js
│   ├── ElementSearchDialog
│   │   ├── element-search-dialog.d.ts
│   │   ├── element-search-dialog.js
│   │   ├── equipment-item.d.ts
│   │   ├── equipment-item.js
│   │   ├── index.d.ts
│   │   ├── index.js
│   │   ├── tag-renderer.d.ts
│   │   └── tag-renderer.js
│   ├── FlatParameters
│   │   ├── FlatParameters.d.ts
│   │   ├── FlatParameters.js
│   │   ├── index.d.ts
│   │   └── index.js
│   ├── Login
│   │   ├── index.d.ts
│   │   ├── index.js
│   │   ├── Login.d.ts
│   │   ├── Login.js
│   │   ├── Logout.d.ts
│   │   └── Logout.js
│   ├── MuiVirtualizedTable
│   │   ├── ColumnHeader.d.ts
│   │   ├── ColumnHeader.js
│   │   ├── index.d.ts
│   │   ├── index.js
│   │   ├── KeyedColumnsRowIndexer.d.ts
│   │   ├── KeyedColumnsRowIndexer.js
│   │   ├── MuiVirtualizedTable.d.ts
│   │   └── MuiVirtualizedTable.js
│   ├── MultipleSelectionDialog
│   │   ├── index.d.ts
│   │   ├── index.js
│   │   ├── MultipleSelectionDialog.d.ts
│   │   └── MultipleSelectionDialog.js
│   ├── OverflowableText
│   │   ├── index.d.ts
│   │   ├── index.js
│   │   ├── overflowable-text.d.ts
│   │   └── overflowable-text.js
│   ├── react-hook-form
│   │   ├── autocomplete-input.d.ts
│   │   ├── autocomplete-input.js
│   │   ├── booleans
│   │   │   ├── boolean-input.d.ts
│   │   │   ├── boolean-input.js
│   │   │   ├── checkbox-input.d.ts
│   │   │   ├── checkbox-input.js
│   │   │   ├── switch-input.d.ts
│   │   │   └── switch-input.js
│   │   ├── directory-items-input.d.ts
│   │   ├── directory-items-input.js
│   │   ├── error-management
│   │   │   ├── error-input.d.ts
│   │   │   ├── error-input.js
│   │   │   ├── field-error-alert.d.ts
│   │   │   ├── field-error-alert.js
│   │   │   ├── mid-form-error.d.ts
│   │   │   └── mid-form-error.js
│   │   ├── ExpandingTextField.d.ts
│   │   ├── ExpandingTextField.js
│   │   ├── numbers
│   │   │   ├── float-input.d.ts
│   │   │   ├── float-input.js
│   │   │   ├── integer-input.d.ts
│   │   │   ├── integer-input.js
│   │   │   ├── utils.d.ts
│   │   │   └── utils.js
│   │   ├── radio-input.d.ts
│   │   ├── radio-input.js
│   │   ├── raw-read-only-input.d.ts
│   │   ├── raw-read-only-input.js
│   │   ├── select-input.d.ts
│   │   ├── select-input.js
│   │   ├── slider-input.d.ts
│   │   ├── slider-input.js
│   │   ├── text-input.d.ts
│   │   ├── text-input.js
│   │   └── utils
│   │       ├── cancel-button.d.ts
│   │       ├── cancel-button.js
│   │       ├── field-label.d.ts
│   │       ├── field-label.js
│   │       ├── functions.d.ts
│   │       ├── functions.js
│   │       ├── submit-button.d.ts
│   │       ├── submit-button.js
│   │       ├── text-field-with-adornment.d.ts
│   │       └── text-field-with-adornment.js
│   ├── ReportViewer
│   │   ├── filter-button.d.ts
│   │   ├── filter-button.js
│   │   ├── index.d.ts
│   │   ├── index.js
│   │   ├── log-report.d.ts
│   │   ├── log-report-item.d.ts
│   │   ├── log-report-item.js
│   │   ├── log-report.js
│   │   ├── log-table.d.ts
│   │   ├── log-table.js
│   │   ├── multi-select-list.d.ts
│   │   ├── multi-select-list.js
│   │   ├── report-item.d.ts
│   │   ├── report-item.js
│   │   ├── report-tree-view-context.d.ts
│   │   ├── report-tree-view-context.js
│   │   ├── report-viewer.d.ts
│   │   └── report-viewer.js
│   ├── ReportViewerDialog
│   │   ├── index.d.ts
│   │   ├── index.js
│   │   ├── report-viewer-dialog.d.ts
│   │   └── report-viewer-dialog.js
│   ├── SignInCallbackHandler
│   │   ├── index.d.ts
│   │   ├── index.js
│   │   ├── SignInCallbackHandler.d.ts
│   │   └── SignInCallbackHandler.js
│   ├── SilentRenewCallbackHandler
│   │   ├── index.d.ts
│   │   ├── index.js
│   │   ├── SilentRenewCallbackHandler.d.ts
│   │   └── SilentRenewCallbackHandler.js
│   ├── SnackbarProvider
│   │   ├── index.d.ts
│   │   ├── index.js
│   │   ├── SnackbarProvider.d.ts
│   │   └── SnackbarProvider.js
│   ├── TopBar
│   │   ├── AboutDialog.d.ts
│   │   ├── AboutDialog.js
│   │   ├── GridLogo.d.ts
│   │   ├── GridLogo.js
│   │   ├── index.d.ts
│   │   ├── index.js
│   │   ├── TopBar.d.ts
│   │   └── TopBar.js
│   ├── translations
│   │   ├── card-error-boundary-en.d.ts
│   │   ├── card-error-boundary-en.js
│   │   ├── card-error-boundary-fr.d.ts
│   │   ├── card-error-boundary-fr.js
│   │   ├── common-button-en.d.ts
│   │   ├── common-button-en.js
│   │   ├── common-button-fr.d.ts
│   │   ├── common-button-fr.js
│   │   ├── directory-items-input-en.d.ts
│   │   ├── directory-items-input-en.js
│   │   ├── directory-items-input-fr.d.ts
│   │   ├── directory-items-input-fr.js
│   │   ├── element-search-en.d.ts
│   │   ├── element-search-en.js
│   │   ├── element-search-fr.d.ts
│   │   ├── element-search-fr.js
│   │   ├── equipment-search-en.d.ts
│   │   ├── equipment-search-en.js
│   │   ├── equipment-search-fr.d.ts
│   │   ├── equipment-search-fr.js
│   │   ├── flat-parameters-en.d.ts
│   │   ├── flat-parameters-en.js
│   │   ├── flat-parameters-fr.d.ts
│   │   ├── flat-parameters-fr.js
│   │   ├── inputs-en.d.ts
│   │   ├── inputs-en.js
│   │   ├── inputs-fr.d.ts
│   │   ├── inputs-fr.js
│   │   ├── login-en.d.ts
│   │   ├── login-en.js
│   │   ├── login-fr.d.ts
│   │   ├── login-fr.js
│   │   ├── multiple-selection-dialog-en.d.ts
│   │   ├── multiple-selection-dialog-en.js
│   │   ├── multiple-selection-dialog-fr.d.ts
│   │   ├── multiple-selection-dialog-fr.js
│   │   ├── report-viewer-en.d.ts
│   │   ├── report-viewer-en.js
│   │   ├── report-viewer-fr.d.ts
│   │   ├── report-viewer-fr.js
│   │   ├── table-en.d.ts
│   │   ├── table-en.js
│   │   ├── table-fr.d.ts
│   │   ├── table-fr.js
│   │   ├── top-bar-en.d.ts
│   │   ├── top-bar-en.js
│   │   ├── top-bar-fr.d.ts
│   │   ├── top-bar-fr.js
│   │   ├── treeview-finder-en.d.ts
│   │   ├── treeview-finder-en.js
│   │   ├── treeview-finder-fr.d.ts
│   │   └── treeview-finder-fr.js
│   └── TreeViewFinder
│       ├── index.d.ts
│       ├── index.js
│       ├── TreeViewFinder.d.ts
│       └── TreeViewFinder.js
├── hooks
│   ├── useDebounce.d.ts
│   ├── useDebounce.js
│   ├── useIntlRef.d.ts
│   ├── useIntlRef.js
│   ├── useSnackMessage.d.ts
│   └── useSnackMessage.js
├── index.d.ts
├── index.js
└── utils
    ├── actions.d.ts
    ├── actions.js
    ├── algos.d.ts
    ├── algos.js
    ├── AuthService.d.ts
    ├── AuthService.js
    ├── ElementType.d.ts
    ├── ElementType.js
    ├── EquipmentType.d.ts
    ├── EquipmentType.js
    ├── styles.d.ts
    ├── styles.js
    ├── UserManagerMock.d.ts
    └── UserManagerMock.js

25 directories, 208 files
```
</details>

Reply:
Did that mean that restore the possibility for the imports ? (like `import ... from 'commons-ui/something`)

Reply:
Damn you're right me neither. :facepalm:

Reply:
> Did that mean that restore the possibility for the imports ? (like `import Abc from '@gridsuite/commons-ui/abc';`)

Yes it should but still not recommanded